tests/trace-viewer.spec.ts

```diff
@@ -94,9 +102,13 @@ test('should refresh trace viewer while test is running', async ({ activate, cre
   });
 
   await testController.expandTestItems(/test.spec/);
-  selectTestItem(testController.findTestItems(/pass/)[0]);
+  const [testItem] = testController.findTestItems(/pass/);
+  selectTestItem(testItem);
 
   const testRunPromise = testController.run();
+  // wait for test to start
+  await waitForTestItemStatus(testItem, 'started');
```

Comment:
sounds like the next line should take care of it since you are polling?

Reply:
My problem is that the polling fails with timeout, at least in my computer, and that's because tests are taking more than 5 seconds to start.

That `waitForTestItemStatus(testItem, 'started')` is just to ensure test is in a state where I have confidence I won't get a timeout while polling.